main/options/options.cc

```diff
@@ -154,6 +154,10 @@ vector<reference_wrapper<PrinterConfig>> Printers::printers() {
     });
 }
 
+bool Printers::isAutogenPrintingDepDB() const {
```

Comment:
It's pretty confusing that it's even possible for `isAutogen && !isAutogenPrintingDepDB` to be true

Reply:
Yeah, it is. That's just a quirk of how we configure things though -- I suppose I could do a bit of refactoring and better variable naming to make things a bit more clear?